server/shared/database/seed/index.js

```diff
@@ -0,0 +1,115 @@
+'use strict'
+
+const Promise = require('bluebird')
+const pick = require('lodash.pick')
+
+const userData = require('./users.json').data
+const dbConnectionData = require('./dbConnections.json').data
+const examData = require('./exams.json').data
+const questionData = require('./questions.json').data
+
+const DIFFICULTY = ['EASY', 'MEDIUM', 'HARD']
+
+function insertAnswers(Answer, examsTaken) {
+  return Promise.map(examsTaken, examTaken => {
+    return examTaken.getStudent().then(student => {
+      return examTaken.getQuestions().then(questions => {
```

Comment:
`.getStudent()` and `.getQuestions()` can be a part of one `Promise.all()` call.